src/ui/components/DropdownMenu/DropdownMenuItem.js

```diff
@@ -0,0 +1,49 @@
+/* @flow */
+import React from 'react';
+import classNames from 'classnames';
+
+import Link from 'amo/components/Link';
+
+
+type Props = {|
+  title: string,
+  to?: string,
+  onClick?: Function,
+  detached?: boolean,
+|};
+
+const DropdownMenuItem = ({ title, to, onClick, detached = false }: Props) => {
+  if (to && onClick) {
+    throw new Error('You cannot set both `to` and `onClick` props');
+  }
+
+  if (!to && !onClick) {
+    return (
+      <li className="DropdownMenuItem DropdownMenuItem-section">
+        {title}
+      </li>
+    );
+  }
+
+  const classnames = classNames('DropdownMenuItem-link', {
+    'DropdownMenuItem-link--detached': detached,
+  });
+
+  if (to) {
+    return (
+      <li className="DropdownMenuItem">
+        <Link className={classnames} to={to}>{title}</Link>
+      </li>
+    );
+  }
+
+  return (
+    <li className="DropdownMenuItem">
+      <button className={classnames} onClick={onClick}>
```

Comment:
I wanted a simple clickable item, but I guess I could use the existing `Button` and unstyle it. Also, I read this issue: https://github.com/mozilla/addons-frontend/issues/1904. Should I do add `href="#"` too?

Reply:
I think this is fine then, actually. Our button component is as much about UI/style as it is a button. I think you can leave this as-is... provided you tested it on Firefox for Android and the first click works. Because yeah otherwise you'll want something that eventually outputs an `<a href="#">` because of the strangeness of the first/second click 😒